src/core/Entry.cpp

```diff
@@ -192,8 +192,10 @@ QString Entry::tags() const
 
 QStringList Entry::tagList() const
 {
-    static QRegExp rx("(\\ |\\,|\\.|\\:|\\t|\\;)");
-    return tags().split(rx, QString::SkipEmptyParts);
+    static QRegExp rx("(\\ |\\,|\\t|\\;)");
+    auto taglist = tags().split(rx, QString::SkipEmptyParts);
+    std::sort(taglist.begin(), taglist.end());
```

Comment:
This sorts the tags in the entry. I suppose this affects the preview panel as well (if not, it should)?

The other place is the tags list in the side bar. Those should be sorted too, if not already done. The automatic tags (searches etc) there should be sorted separately from the manually-created ones. And probably keep "All" at top of everything.

Reply:
This is all done already.